src/status_im2/contexts/wallet/common/temp.cljs

```diff
@@ -73,3 +62,11 @@
    :account-name        "Account 1"
    :account             :default
    :customization-color :blue})
+
+(def create-account-state
+  [{:title        "Alisher's default keypair"
```

Comment:
can we get this title as a default? 🤔  like user name + default keypair?

Reply:
@J-Son89 fixed